desk/app/activity.hoon

```diff
@@ -502,8 +612,10 @@
   (refresh-index source index(stream new) |)
 ++  refresh-index
   |=  [=source:a new=index:a new-floor=?]
+  ~?  verbose  "refeshing index: {<source>}"
```

Comment:
You may consider prefixing all the verbose prints with `%{(trip dap.bowl)}`, for log reading clarity.

(Arguably at that point there's enough boilerplate to want to do a `+log` arm such that you `%-  (log "refreshing index")  rest-of-the-hoon`, but the `~?` are also fine for now.)